components/clarinet-cli/src/chainhooks/types.rs

```diff
@@ -1,433 +0,0 @@
-use serde::{Deserialize, Serialize};
```

Comment:
This file wasn't used, so I took the opportunity to delete it

Reply:
Good call, thanks!